source/run-command.txt

```diff
@@ -0,0 +1,182 @@
+.. _c-run-command:
+
+======================
+Run a Database Command
```

Comment:
[note to reviewer] Added "Database" to this title as this page will live in the `/databases-collections` drawer.